packages/o/openshadinglanguage/package.yml

```diff
@@ -9,21 +9,30 @@ component  : multimedia.graphics
 summary    : Advanced shading language for production GI renderers
 description: |
     Open Shading Language (OSL) is a small but rich language for programmable shading in advanced renderers and other applications, ideal for describing materials, lights, displacement, and pattern generation.
-clang      : yes
 builddeps  :
     - pkgconfig(OpenEXR)
     - pkgconfig(OpenImageIO)
     - pkgconfig(Qt5Core)
     - pkgconfig(pugixml)
     - pkgconfig(python3)
     - libboost-devel
+    - llvm-clang-15-devel
     - numpy
     - partio-devel
     - pybind11
 rundeps    :
     - numpy
+environment: |
+    export CMAKE_PREFIX_PATH=/usr/lib64/llvm15
```

Comment:
Can you use some variation of `llvm-config-15 --cmakedir` instead? That would be more robust if it ever gets moved around.

Reply:
Oh interesting, thanks for the note, I didn't know this exists! I think `llvm-config-15 --prefix` would indeed work. Let me try replacing it and see if anything blows up...

Reply:
That works! :tada: This does indeed feel much better than hard-coding the path!